Gemfile.lock

```diff
@@ -47,7 +47,7 @@ GEM
       multi_json (~> 1.0)
     addressable (2.3.5)
     arel (3.0.3)
-    bcrypt (3.1.7)
+    bcrypt (3.1.10)
```

Comment:
This is a small thing, but given Slices is open source maybe we should be a bit more careful than otherwise: shall we avoid updating other gems for now (i.e. in this commit)? The problem is you run `bundle update foo` and anything in the dependency chain that is not version locked in the `Gemfile` gets updated. So just updating/installing `will_paginate_mongoid` will cause a mongoid version bump, which causes a moped version bump, etc. Obviously they are just patch-level version changes so should be fine to put in, but why take the risk? If the point of the commit is to change will_paginate, and that change can be accomplished without updating the other gems (i.e. will_paginate does not _require_ mongoid 3.1.7 specifically), then leave them be and keep the change as small as possible.